djmoney/serializers.py

```diff
@@ -25,7 +23,7 @@ def Deserializer(stream_or_string, **options):  # noqa
 
     ignore = options.pop("ignorenonexistent", False)
 
-    if not isinstance(stream_or_string, (bytes, six.string_types)):
```

Comment:
It seemed we had a bug - `string_types` is a tuple, which makes the check incorrect :) 
Currently on Python 3:

```
(bytes, (str,))
```

but after the change it is correct :)

Reply:
Oh yeah you're right, haha. But somehow it still works.. I can't explain:

    >>> isinstance("test", (bytes, (str,),))
    True

Reply:
Hm, it is strange :) I didn't know that it works this way. It is working even for more nesting :))

```
>>> isinstance(1.0, (bytes, (int, (float, ))))
True
```